src/Inciter/AMR/edge.cpp

```diff
@@ -0,0 +1,9 @@
+#include "edge.h"
+
+// Implementation of the friend function, extending osstream and not an edge_t
+// member
+std::ostream& operator<<(std::ostream& os, const edge_t& e)
+{
+    os << e.data.first << "-" << e.data.second;
+    return os;
+}
```

Comment:
Can we use `.C` as the extension? Just to make it consistent with the rest of the code (for better or worse).